pkgs/by-name/zo/zotify/package.nix

```diff
@@ -0,0 +1,46 @@
+{
+  fetchFromGitHub,
+  python3Packages,
+  lib,
+}:
+python3Packages.buildPythonApplication rec {
+  pname = "zotify";
+  version = "0.16.13";
+
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "zotify-dev";
+    repo = "zotify";
+    # repository has no version tags
+    rev = "5da27d32a1f522e80a3129c61f939b1934a0824a";
+    hash = "sha256-KA+Q4sk+riaFTybRQ3aO5lgPg4ECZE6G+By+x2uP/VM=";
+  };
+
+  build-system = [ python3Packages.setuptools ];
+
+  nativeBuildInputs = [ python3Packages.pythonRelaxDepsHook ];
+
+  pythonRelaxDeps = [ "protobuf" ];
+
+  dependencies = with python3Packages; [
+    ffmpy
+    music-tag
+    pillow
+    tabulate
+    tqdm
+    librespot
+    pwinput
+    protobuf
+  ];
+
+  doCheck = false;
```

Comment:
Why do you disable tests here?

Also, please add `pythonImportsCheck` for a simple check.
https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#using-pythonimportscheck-using-pythonimportscheck

Reply:
There were no tests, but I guess I shouldn't have explicitly disabled it. I added `pythonImportsCheck` now.